sbin/health-checker.in

```diff
@@ -10,24 +10,12 @@
 # - if reboot does not help, stop system before further damage is done
 #
 
-STATE_FILE=/var/lib/misc/health-check.state
-REBOOTED_STATE=/var/lib/misc/health-check.rebooted
 PLUGINDIR=@PLUGINDIR@
 USR_LOCAL_PLUGINDIR=/usr/local/libexec/health-checker
+ETC_PLUGINDIR=/etc/health-checker
```

Comment:
Please, don't add scripts to /etc, the mid- to long term plan is still to make /etc non-executable. Only remaining problem is grub...
And why do we need /etc/health-checker?
If somebody wants to provide plugins via RPM, /usr/libexec/health-checker is the correct location.
If somebody wants to use something else, /usr/local/libexec/health-checker is the correct place.
I don't see a need for /etc/health-checker.

Reply:
Thanks for the explanation Thorsten. `/etc/health-checker` has been added just for testing purposes because the virtual machines I am generating on OBS have some issues with `/usr/local/libexec/health-checker`. They don't work with `/etc/health-checker` regardless, so I'll remove the new path and try the official images. I should have added a comment regarding the testing, my bad.